### DISCUSSION TITLE
HTML video not working on app

### DISCUSSION THREAD

**Discussion by DianaSamayoa** [rocksdanister/lively#1540] [Q&A] [answered]
Hi!! new here. I tried to make a custom WP simple with a video and the OpenWeather function using the cyberpunk wallpaper as base
In chrome everything looks fine when testing the html code
but at the moment to put the zip on the app the Wallpaper doesn't play the video, it stays gray. I don't know how to program in html...so I don't see if I'm doing something wrong also the dimensions look weird but I'm more concerned about the playing issue but if you know a  solution that would be great. Sorry for bad english. 
here is my file
[-Nier2.zip](https://github.com/rocksdanister/lively/files/10502188/-Nier2.zip)

**Comment by rocksdanister** [comment, marked as answer]
Its a codec license issue, convert the video file to vp9 webm file:
https://github.com/rocksdanister/lively/wiki/Web-Guide-VI-:-Webpage-Video-Support

**Reply by DianaSamayoa** [reply]
thank you so much it works perfectly! :D 
I don't make web dev so... I always forget about the webm files existence
